frontend/src/apiAccess/apiApps.js

```diff
@@ -184,6 +186,11 @@ export const apiReduxApps = {
     responseEntityType: "cards",
   }),
 
+  FETCH_USER_BURNDOWN_SNAPSHOTS_PAGE: createReduxApp({
+    BASE_TYPE: FETCH_USER_BURNDOWN_SNAPSHOTS_PAGE,
+    apiCaller: apiCallers.userBurndownSnapshotsPage,
```

Comment:
```
    responseIsList: true, // since we are fetching a list of things
    responseEntityType: "burndownSnapshot", // we need this to be stored in the state
```